apps/fluent-tester/src/E2E/Callout/specs/Callout.spec.win.ts

```diff
@@ -19,26 +21,40 @@ describe('Callout Testing Initialization', function () {
   });
 });
 
-/* This will be re-enabled after the Callout's ViewManager integrates the testID prop. This is used to select UI components */
-// describe('Callout Functional Testing', () => {
-//   it('Open the callout and validate it loaded correctly (visible)', () => {
-//     CalloutPageObject.scrollToTestElement();
-//     CalloutPageObject.openCallout();
-//     CalloutPageObject.waitForCalloutComponentInView(PAGE_TIMEOUT);
-//     expect(CalloutPageObject.didCalloutLoad()).toBeTruthy();
-//   });
-// });
-
-// describe('Callout Accessibility Testing', () => {
-//   it('Validate accessibilityRole is correct', () => {
-//     CalloutPageObject.scrollToTestElement();
-//     CalloutPageObject.waitForCalloutComponentInView(PAGE_TIMEOUT);
-//     expect(CalloutPageObject.getAccessibilityRole()).toEqual(CALLOUT_A11Y_ROLE);
-//   });
-
-//   it('Set accessibilityLabel', () => {
-//     CalloutPageObject.scrollToTestElement();
-//     CalloutPageObject.waitForCalloutComponentInView(PAGE_TIMEOUT);
-//     expect(CalloutPageObject.getAccessibilityLabel(CalloutSelector.Primary)).toEqual(CALLOUT_ACCESSIBILITY_LABEL);
-//   });
-// });
+describe('Callout Accessibility Testing', () => {
+  beforeAll(async () => {
+    await CalloutPageObject.scrollToTestElement();
+    await CalloutPageObject.openCallout();
+    await CalloutPageObject.waitForCalloutComponentInView(PAGE_TIMEOUT);
+  });
+
+  it('Validate accessibilityRole is correct', async () => {
+    await expect(await CalloutPageObject.getAccessibilityRole()).toEqual(CALLOUT_A11Y_ROLE);
+    await expect(await CalloutPageObject.didAssertPopup()).toBeFalsy(CalloutPageObject.ERRORMESSAGE_ASSERT); // Ensure no asserts popped up
+  });
+
+  it('Set accessibilityLabel', async () => {
```

Comment:
Does a Callout actually have an accessibility label on it? Or is it on the items inside the callout?

What's normal usage? @PPatBoyd Is it necessary to have this test?

Reply:
TBH I'm a little unsure from memory, @ksiler do you recall better the usage of accessibilityLabel relative to a Callout? I believe we need it because the window does get added to the accessibility tree as uhhh unsure what control type but can find out, but I'm less clear on when the label would be used (what level of verbosity etc when a callout appears)

Reply:
Bump

Reply:
I think the control type is "Group" on win32 but I don't recall if an a11y label is needed or not.

Reply:
I would think just the items in the callout would need to be accessible but I'm unsure how someone would place keyboard focus within the callout if there's focusable items in it. Does focus automatically get moved there? And you escape it via "Esc"?

Reply:
Generally `Esc` will close any callout, yes (barring host app interactions).  With regards to moving focus there that may not currently be testable with rex-win32, and generally indicates we're in special territory that also relies on host app interactions to place focus inside the callout (this is relatively win32-specific as far as I know) e.g. some applications use `f6` as a navigation key